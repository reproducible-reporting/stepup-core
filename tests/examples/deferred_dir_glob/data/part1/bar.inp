Bar input of part 1
