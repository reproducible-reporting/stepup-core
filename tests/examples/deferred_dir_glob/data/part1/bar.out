Bar output of part 1
